backend/src/main/java/ch/puzzle/okr/dto/keyresult/KeyResultLastCheckInMetricDto.java

```diff
@@ -3,5 +3,5 @@
 import java.time.LocalDateTime;
 
 public record KeyResultLastCheckInMetricDto(Long id, Double value, Integer confidence, LocalDateTime createdOn,
-        String comment) implements KeyResultLastCheckIn {
+        String changeInfo, String initiatives) implements KeyResultLastCheckIn {
```

Comment:
Wieso das Feld `initiatives`? Das finde ich in der Rest-Schnittstelle nicht.

Reply:
Das wurde im Nachhinein von @peggimann neu vorgegeben, siehe #307